src/main/java/org/mockito/MockitoFramework.java

```diff
@@ -92,4 +92,55 @@
      */
     @Incubating
     InvocationFactory getInvocationFactory();
+
+    /**
+     * Clears up internal state of all inline mocks.
+     * This method is only meaningful if inline mock maker is in use.
+     * Otherwise this method is a no-op and need not be used.
+     * <p>
+     * This method is useful to tackle subtle memory leaks that are possible due to the nature of inline mocking
+     * (issue <a href="https://github.com/mockito/mockito/pull/1619">#1619</a>).
+     * If you are facing those problems, call this method at the end of the test (or in "@After" method).
```

Comment:
Shouldn't we update our JUnit runner and extension to handle this automatically? I think this is an improvement in any situation, so we can just go ahead and do it eagerly.

Reply:
This would be a breaking change in thread-based parallel tests that are supported by Mockito (with limitations, as we stated in the FAQ).

If you turn this on by default, some Mockito tests will start failing. @ttanxu, discussed this somewhere in this PR (but the thread became long :).

We can always make it later if we understand the use patterns better. I suggest we expose the this API first and observe / learn how it goes.

Reply:
Ah okay, I missed that discussion. Thanks for the clarification.